src/Ontology/Account.h

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <array>
+#include <vector>
+#include <string>
+
+#include "Address.h"
+#include "PrivateKey.h"
+#include "BinaryCoding.h"
+
+namespace TW {
+namespace Ontology {
+
+class Account {
```

Comment:
I think this one is used to sign the transaction? probably name it as `Signer`

Reply:
I want to know if I change it to `Signer`, whether it will make other developer feels confusing with `TWOntologySigner`. If not, I will change it's name to `Signer`.

Reply:
Ok, What `Account` actual does is trivial (just holds a PrivateKey), I wonder if we need this kind of abstraction,  and there is another `sign` method in `Transaction` which looks unusual. In general, we use a signer to `sign` the transaction, not a transaction to `sign` an account.

Reply:
I agree with your opinion, I will change its class name.

Reply:
Here is the `Signer` class with `sign`, `addSign` method. https://github.com/TrustWallet/wallet-core/pull/201/commits/da6edb41083bb71a8cf243933bc85c49870c0255